test/api/groups.coffee

```diff
@@ -151,7 +151,7 @@ describe "Guilds", ->
           done()
 
   context "removing users groups", ->
-    it "allows guild leaders to remove a member", (done) ->
+    it "allows guild leaders to remove a member (but not themselves)", (done) ->
```

Comment:
@crookedneighbor I've modified a test to be sure a change is working, it's ok to modify it or you prefer a new test to be created?